katello/katello/backup.rb

```diff
@@ -329,7 +336,7 @@ def create_pulp_data_tar
 
     def backup_config_files
       puts "Backing up config files... "
-      run_cmd("tar --selinux --create --gzip --file=#{File.join(@dir, 'config_files.tar.gz')} --listed-incremental=#{File.join(@dir, '.config.snar')} #{configure_configs.join(' ')} 2>/dev/null", [0,2])
+      run_cmd("tar --selinux --create --gzip --file=#{File.join(@dir, 'config_files.tar.gz')} --listed-incremental=#{File.join(@dir, '.config.snar')} #{configure_configs.join(' ')} #{excluded_configs.join(' ')} 2>/dev/null", [0,2])
```

Comment:
This won't work, `--exclude` needs to be positioned before listing the files or directories you want to backup.